README.md

```diff
@@ -300,6 +283,8 @@ You can try the `API_BUSINESS_PARTNER` service with a real S/4HANA system with t
     cds add connectivity
     ```
 
+5. `cds add helm` will not add configuration required to create a Connectivity Service Instance. This Service Instance should be created by the Kyma Cluster Administrator. For more information regarding configuration of Connectivity Instance, please check the [documentation](https://cap.cloud.sap/docs/guides/deployment/deploy-to-kyma#connectivity-service).
```

Comment:
Should this rather be part of step 4. A connectivity service instance seems to be only required for on-premise

Reply:
yeah. I added it as a Note to step 4.